state/accumulator.go

```diff
@@ -423,7 +423,7 @@ func (a *Accumulator) Accumulate(txn *sqlx.Tx, userID, roomID string, prevBatch
 					redactsEventID = parsedEv.Get("content.redacts").Str
 				}
 				if redactsEventID != "" {
-					redactTheseEventIDs = append(redactTheseEventIDs, redactsEventID)
+					redactTheseEventIDs[redactsEventID] = &dedupedEvents[i]
```

Comment:
This is a map from redactee to redaction. It's possible that two people concurrently send redactions for the same event and they both appear in the timeline. Does this matter? I would assume not: redactions are basically idempotent, and we should set `redacted_because` to something so clients will be happy.

Reply:
Yeah last write wins then.